include/toml++/impl/preprocessor.h

```diff
@@ -399,7 +399,27 @@
 
 // dllexport etc
 #ifndef TOML_API
-	#define TOML_API
+	#if TOML_HEADER_ONLY == 0
```

Comment:
```suggestion
	#if !TOML_HEADER_ONLY
```

Just to be sylistically consistent with the other preprocessor stuff in the library